src/block-management/block-manager.js

```diff
@@ -263,8 +250,8 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 			list = (
 				<FlatList
 					style={ styles.list }
-					data={ this.props.blocks }
-					extraData={ this.props.refresh, this.state.inspectBlocks }
```

Comment:
Not sure how it was working before but this looks like it's equivalent with `extraData={ this.state.inspectBlocks }`